xbmc/Application.cpp

```diff
@@ -3724,6 +3724,10 @@ bool CApplication::OnMessage(CGUIMessage& message)
         ShowAppMigrationMessage();
 
         m_bInitializing = false;
+
+        if (message.GetParam1() == GUI_MSG_UI_READY_PROFILE)
```

Comment:
How about `if (message.GetSenderId() == WINDOW_SETTINGS_PROFILES)`? We should avoid adding another GUI message if possible.

Reply:
I'll test that.

Reply:
You'll need to set `WINDOW_SETTINGS_PROFILES` in the profiles code.